deno/rpc/common.ts

```diff
@@ -31,3 +31,117 @@ export enum RPCCloseEventCodes {
 	InvalidVersion,
 	InvalidEncoding,
 }
+
+/**
+ * https://discord.com/developers/docs/topics/rpc#commands-and-events-rpc-commands
+ */
+export enum RPCCommands {
+	DISPATCH = 'DISPATCH',
+	AUTHORIZE = 'AUTHORIZE',
+	AUTHENTICATE = 'AUTHENTICATE',
+	GET_GUILD = 'GET_GUILD',
+	GET_GUILDS = 'GET_GUILDS',
+	GET_CHANNEL = 'GET_CHANNEL',
+	GET_CHANNELS = 'GET_CHANNELS',
+	SUBSCRIBE = 'SUBSCRIBE',
+	UNSUBSCRIBE = 'UNSUBSCRIBE',
+	SET_USER_VOICE_SETTINGS = 'SET_USER_VOICE_SETTINGS',
+	SELECT_VOICE_CHANNEL = 'SELECT_VOICE_CHANNEL',
+	GET_SELECTED_VOICE_CHANNEL = 'GET_SELECTED_VOICE_CHANNEL',
+	SELECT_TEXT_CHANNEL = 'SELECT_TEXT_CHANNEL',
+	GET_VOICE_SETTINGS = 'GET_VOICE_SETTINGS',
+	SET_VOICE_SETTINGS = 'SET_VOICE_SETTINGS',
+	SET_CERTIFIED_DEVICES = 'SET_CERTIFIED_DEVICES',
+	SET_ACTIVITY = 'SET_ACTIVITY',
+	SEND_ACTIVITY_JOIN_INVITE = 'SEND_ACTIVITY_JOIN_INVITE',
+	CLOSE_ACTIVITY_JOIN_REQUEST = 'CLOSE_ACTIVITY_JOIN_REQUEST',
+	/* https://github.com/discordjs/RPC/blob/master/src/constants.js#L14 */
```

Comment:
Probably would help to either
1. get this documented on discord
2. Label this as unstable (see other examples in the module)